packages/model-eval/src/discovery.config.ts

```diff
@@ -0,0 +1,73 @@
+import {
+  EvalConfig,
+  makeMongoDbCommandMetadataStore,
+  makeMongoDbGeneratedDataStore,
+  makeMongoDbEvaluationStore,
+  makeMongoDbReportStore,
+  getConversationsTestCasesFromYaml,
+} from "mongodb-chatbot-evaluation";
+import fs from "fs";
+import path from "path";
+import "dotenv/config";
+import { MongoClient, assertEnvVars } from "mongodb-rag-core";
+import { envVars } from "./envVars";
+import { makeChatLlmConversationEvalCommands } from "./makeChatLlmEvalCommands";
+import { makeRadiantChatLlm } from "./makeRadiantChatLlm";
+import { radiantModels } from "./radiantModels";
+
+export default async () => {
```

Comment:
Seems like there's a good deal of overlap here with `quizEval.config.ts` - maybe worth sharing a common base config?

```ts
{
  ...baseModelEvalConfig,
  commands: makeChatLlmConversationEvalCommands({
    chatLlmConfigs,
    testCases: discoveryConversationTestCases,
  }),
}
```

or maybe even combining into a single config, e.g.

```ts
{
  commands: {
    ...makeChatLlmConversationEvalCommands({
      chatLlmConfigs,
      testCases: discoveryConversationTestCases,
    }),
    ...makeChatLlmQuizEvalCommands({
      configs: modelsConfig,
      quizQuestions: quizQuestionTestCases,
    })
  }
}
```

Reply:
good point. will make a common base